app/views/cats/show.html.erb

```diff
@@ -0,0 +1,11 @@
+  <div class="cat-title">
+    <p><%= @cat.name %></p>
+    <p><%= @cat.breed %></p>
+    <p><%= @cat.color %></p>
+    <p><%= @cat.description %></p>
+  <div class="details">
+    <h2>A bit more about this cat</h2>
+    <p><%= @cat.description %></p>
+  </div>
+</div>
+
```

Comment:
could you please add a link_to Booking, so that the booking form gets attached to your page? Thank you :)